client/blocks/app-banner/index.jsx

```diff
@@ -190,9 +237,35 @@ export class AppBanner extends Component {
 				</Card>
 			</div>
 		);
+	};
+
+	render() {
+		if ( ! this.props.shouldDisplayAppBanner || this.state.isDraftPostModalShown ) {
+			return null;
+		}
+
+		const displayJetpackAppBranding = config.isEnabled( 'jetpack/app-branding' );
+
+		return displayJetpackAppBranding
+			? this.getJetpackAppBanner( this.props )
+			: this.getWordpressAppBanner( this.props );
 	}
 }
 
+function BannerIcon( { icon } ) {
+	useEffect( () => {
+		lottie.loadAnimation( {
+			container: document.querySelector( '.app-banner__icon' ),
+			renderer: 'svg',
+			loop: false,
+			autoplay: true,
+			path: icon,
+		} );
```

Comment:
It seems unlikely a memory leak would occur, but it is probably best to destroy the animation in the Hook's clean-up function. 
```suggestion
		const animation = lottie.loadAnimation( {
			container: document.querySelector( '.app-banner__icon' ),
			renderer: 'svg',
			loop: false,
			autoplay: true,
			path: icon,
		} );
		return () => animation.destroy();
```

From looking at the detached nodes in the Chrome DevTools Memory tab, it did appear that destroying the animation caused the detached SVG elements to be released sooner. Again, I see no reason why the current code would cause a memory leak — the elements would likely still be released as no references to them are stored. However, cleaning up is likely still a safe practice worth implementing. 

| Before | After |
| - | - |
| <img width="569" alt="memory-before" src="https://user-images.githubusercontent.com/438664/187778438-3da385ca-62a6-47ca-9de2-42a1da7cb053.png"> | <img width="556" alt="memory-after" src="https://user-images.githubusercontent.com/438664/187778445-6d8face4-8611-480e-9120-f40a3ab6604f.png"> |